README.md

```diff
@@ -27,11 +27,9 @@ The Prototype Kit always supports at least the current and previous LTS releases
 
 ## Support
 
-The GOV.UK Prototype Kit is maintained by the Government Digital Service. If you’ve got a question or need support you can:
+The GOV.UK Prototype Kit is currently maintained by the [GOV.UK Design System team](https://design-system.service.gov.uk/get-in-touch/). There is only [minimal support in place](https://github.com/alphagov/govuk-prototype-kit/issues/2389) as the current version is stable.
```

Comment:
I'm sure we changed this wording elsewhere – I'm hesitant to suggest that because the kit is stable we only 'need' to provide minimal support.

I think it's better to just say something like 'we can only provide minimal support at this time'.

Reply:
Good spot! Changed it 👍🏻